packages/api-explorer/src/block-types/Content.jsx

```diff
@@ -43,12 +43,12 @@ const Loop = ({ content, column, flags, splitReferenceDocs }) => {
 };
 
 const Content = props => {
-  const { body, flags, isThreeColumn, splitReferenceDocs, useNewMarkdownEngine } = props;
+  const { body, copyButtons, flags, isThreeColumn, splitReferenceDocs, useNewMarkdownEngine } = props;
 
   if (useNewMarkdownEngine) {
     const content = markdown(body, {
       compatibilityMode: flags.rdmdCompatibilityMode,
-      copyButtons: false,
+      copyButtons,
```

Comment:
Since this component is used in response schemas that are funneled through Markdown, this is going to add copy buttons to schema descriptions there. Can you set a `false` default to `copyButtons` so the only thing that'll get copy buttons are operation descriptions in the center column?

Reply:
https://github.com/readmeio/api-explorer/pull/1260